app/src/main/java/com/cornellappdev/android/eatery/model/enums/OnboardingPageType.java

```diff
@@ -1,11 +1,28 @@
 package com.cornellappdev.android.eatery.model.enums;
 
+import com.cornellappdev.android.eatery.R;
+
 public enum OnboardingPageType {
     MENUS,
     COLLEGETOWN,
     TRANSACTIONS,
     LOGIN;
 
+    public int getAnimationRaw() {
+        switch (this) {
+            case MENUS:
+                return R.raw.lottie;
+            case COLLEGETOWN:
+                return  R.raw.lottie2;
+            case TRANSACTIONS:
+                return  R.raw.lottie3;
+            case LOGIN:
+                return  R.raw.lottie;
+            default:
+                return  R.raw.lottie;
```

Comment:
please rename these to something more descriptive. also couldn't you do something like 
```
case MENUS, LOGIN:
    return R.raw.lottie;
.....
```

Reply:
![image](https://user-images.githubusercontent.com/43295248/70164961-157f7e00-1690-11ea-917f-023858fe64c9.png)